app/views/compute_resources_vms/form/ovirt/_volume.html.erb

```diff
@@ -5,14 +5,13 @@
              { }, :label => _('Storage domain'), :label_size => "col-md-2", :disabled => !new_vm, :class => "col-md-2" %>
 <%= f.hidden_field :storage_domain if !new_vm %>
 <%= f.hidden_field :id %>
-
-<%= checkbox_f f, :preallocate, { :checked => f.object.sparse == 'false', :help_inline => _('Uses thin provisioning if unchecked'), :label => _('Preallocate disk'), :label_size => "col-md-2" } %>
-<%= checkbox_f f, :wipe_after_delete, { :checked => f.object.wipe_after_delete == 'true', :label => _('Wipe disk after delete'), :label_size => "col-md-2" } %>
+<%= checkbox_f f, :preallocate, { :checked => f.object.sparse == false, :help_inline => _('Uses thin provisioning if unchecked'), :label => _('Preallocate disk'), :label_size => "col-md-2" } %>
+<%= checkbox_f f, :wipe_after_delete, { :checked => f.object.wipe_after_delete , :label => _('Wipe disk after delete'), :label_size => "col-md-2" } %>
 <%= select_f f, :interface, [OpenStruct.new({:id =>'virtio', :name =>'VirtIO'}), OpenStruct.new({:id =>'virtio_scsi', :name =>'VirtIO SCSI'}), OpenStruct.new({:id =>'ide', :name =>'IDE'})],
              :id, :name, { :include_blank => true }, :label => _('Disk interface'), :label_size => "col-md-2", :disabled => !new_vm, :class => "col-md-2" %>
 <%= f.hidden_field :interface if !new_vm %>
 
 <%= field(f, :bootable, :label => _('Bootable'), :label_size => "col-md-2") do
-    radio_button_f f, :bootable, {:disabled => !new_vm, :value=>'true', :checked => (f.object.bootable == 'true'), :onclick => 'tfm.computeResource.ovirt.bootableRadio(this)',
+    radio_button_f f, :bootable, {:disabled => !new_vm, :value=>'true', :checked => f.object.bootable , :onclick => 'tfm.computeResource.ovirt.bootableRadio(this)',
```

Comment:
shouldn't the value also change from `'true'` to `true`?

Reply:
I don't think it matters, I'm not sure what this value spouse to be, but 'true' and true works equally the same.
it's a different input filed then the the fields I've changed so i'm not sure they should be treated the same.
I can deeply investigate on this if you want but I think it should stay as it is.

Reply:
I meant keeping it consistent within the radio button itself, either string or boolean for value and checked.